SpatialGDK/Documentation/content/authority.md

```diff
@@ -59,13 +59,15 @@ Actor on **non-owning client**:
 
 ## GDK authority
 
-As the GDK works with multiple [server-workers]({{urlRoot}}/content/glossary#worker), rather than a single server, authority needs to be dictated by SpatialOS so that authority is shared between server-workers. This means server-workers have authority over some Actors but don’t have authority over other Actors, depending on how SpatialOS assigns authority. **This is a key difference between the GDK and native Unreal networking!**
+As the GDK can work with multiple [server-workers]({{urlRoot}}/content/glossary#worker) using [zoning]({{urlRoot}}/content/glossary#zoning), authority needs to be dictated by SpatialOS so that it is shared between server-workers. This means server-workers have authority over some Actors but don’t have authority over other Actors, depending on how SpatialOS assigns authority. **This is a key difference between the GDK and native Unreal networking!**
 
-> We use the term “authoritative” when a server-worker has authority over an Actor and “non-authoritative” when it doesn’t.
+> **Note:** Support for zoning is currently in pre-alpha. We invite you to try out the [multiserver zoning shooter tutorial]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro) and learn about how it works, but we don’t recommend you start developing features that use zoning yet.
+
+We use the term “authoritative” when a server-worker has authority over an Actor and “non-authoritative” when it doesn’t.
```

Comment:
BTW, I didn't add this - I'm not sure if we want to get rid right now (might take a bit of work to change throughout) or make a ticket.

Reply:
"authoritative" <sobs>
You can imagine my face.

I think this needs a ticket. (It might be a quick-fix for you tomorrow?)

Reply:
Haha! That's why I added the disclaimer :D 
I'll have a go now.

Reply:
Unfortunately "IsAuthoritative" is there in the code, but we can still change it in the docs.